src/main/java/com/code_intelligence/jazzer/utils/Log.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2023 Code Intelligence GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.code_intelligence.jazzer.utils;
+
+import java.io.PrintStream;
+
+/**
+ * Provides static functions that should be used for any kind of output (structured or unstructured)
+ * emitted by the fuzzer.
+ *
+ * <p>Output is printed to {@link System#err} and {@link System#out} until {@link
+ * Log#fixOutErr(PrintStream, PrintStream)} is called, which locks in the {@link PrintStream}s to be
+ * used from there on.
+ */
```

Comment:
Documenting which functions write to `System.out` and which to `System.err` would be helpful.

Reply:
I didn't do this on purpose: We might want to change it if we introduce more structured logging (e.g. to a file or as JSON) or rely more heavily on JUnit mechanisms to report messages. I think that distinguishing by message type (error, finding, structured output, ...) should be sufficient information for the call site.

Do you see a particular benefit in documenting what writes where?

Reply:
No, that clarifies it. Let's leave it as is.